delta-api/src/main/java/io/cdap/delta/api/SourceTable.java

```diff
@@ -21,13 +21,17 @@
 import java.util.List;
 import java.util.Objects;
 import java.util.Set;
+import javax.annotation.Nullable;
 
 /**
  * Object representing what data from a table should be read by the source.
  */
 public class SourceTable {
   private final String database;
   private final String table;
+  // this schema is required for some db to uniquely identify the table, if this is not provided, it will not
+  // be able to fetch the records
+  private final String schema;
```

Comment:
This will also need to be added to 'DBTable' for the describeTable and assessTable endpoints.

we probably need to rethink how a source 'table' is identified as we add more sources.

Reply:
I was thinking to do that last night but didn't do that. Since this schema field is only needed in the `SourceTable` when passing the full table name in Debizium config. Adding this to all table related classes means we also need to add this to `DDLEvent`, etc. And I don't think this is a good way to handle this kind of problem so this pr is like a workaround, maybe I will open a JIRA to track this and fix it in the future

Reply:
Created, https://issues.cask.co/browse/CDAP-16261